projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.ts

```diff
@@ -25,10 +27,16 @@ export class QueryDetailsComponent implements OnChanges{
   private static readonly QUERY_ITEM_FOLLOW_UP = '4';
```

Comment:
Are we using this? Doesn't look like it, and seems to be similar to QueryCreateContext.FOLLOWUP. Probably best to just have one constant